integration/test/native_installer_test.go

```diff
@@ -0,0 +1,35 @@
+//+build install_native
+
+package main
+
+import (
+	"goharvest2/integration/test/installer"
+	"log"
+	"os"
+	"testing"
+)
+
+func TestNativeInstall(t *testing.T) {
+	var path = os.Getenv("BUILD_PATH")
+	if len(path) == 0 {
+		panic("BUILD_PATH variable is not set.")
+	}
+	installObject, error := installer.GetInstaller(installer.NATIVE, path)
+	if error != nil {
+		log.Println("Unable to initialize installer object")
+		panic(error)
+	}
+	if installObject.Install() == true {
```

Comment:
instead of `if installObject.Install() == true {`
use
`if installObject.Install() {`
which maybe points out a problem with the name, should it be 
`DidInstall` or `HasInstalled` or something that communicates a boolean is returned.